src/app.ts

```diff
@@ -46,6 +60,23 @@ app.use(validator({
   schemaPath: path.join(__dirname, '../node_modules/@badgateway/tt-types/schema')
 }));
 
+// a12n setup
+const client = new OAuth2Client({
+  server: process.env.AUTH_API_URI,
+  clientId: process.env.OAUTH2_CLIENT_ID || 'tt-api',
+  clientSecret: process.env.OAUTH2_CLIENT_SECRET,
+});
+
+app.use(browserToBearer({client}));
+
+app.use(oauth2({
+  publicPrefixes: [
+    '/health-check',
```

Comment:
Our standard endpoint for this is `/health`, but looks good otherwise!

```suggestion
    '/health',
```